### PR TITLE
Feature/basic retrieving detail view

### PR DESCRIPTION
**Pull request by AlbertoCastellon**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { NgModule }             from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { CarsComponent }      from './cars/cars.component';
+import { CarDetailComponent }  from './car-detail/car-detail.component';
+
+const routes: Routes = [
+  { path: '', redirectTo: '/cars', pathMatch: 'full' },
+  { path: 'detail/:id', component: CarDetailComponent },
```
Comment (javieraviles) [inline src/app/app-routing.module.ts]: wouldn't make sense `cars/:id` instead of `detail/:id`?

File: src/app/car.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export class Car {
+
+    id: string
+    brand: string
+    registration: Date
+    country: string
+    created_at: Date
```
Comment (javieraviles) [inline src/app/car.ts]: why are created_at and last_updated part of the frontend model?

File: src/app/car.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export class Car {
+
+    id: string
```
Comment (javieraviles) [inline src/app/car.ts]: id won't always be informed, so should be optional (id?: string;)

File: src/app/cars/cars.component.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+<a routerLink="/new-car">
+  <td><button class="New Car ml-4" >New Car</button>
+  </a>
+</div>
+
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th scope="col" sortable="id" (sort)="onSort($event)">#</th>
```
Comment (javieraviles) [inline src/app/cars/cars.component.html]: where is onSort() implemented?

**Review by javieraviles — CHANGES_REQUESTED** [review]
Some changes required

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (AlbertoCastellon) [reply src/app/app-routing.module.ts]: Yes, indeed. It makes more sense. Fixed.

File: src/app/car.ts (hunk 2 above)
Comment (AlbertoCastellon) [reply src/app/car.ts]: Fixed.

File: src/app/car.ts (hunk 3 above)
Comment (AlbertoCastellon) [reply src/app/car.ts]: Fixed.

File: src/app/cars/cars.component.html (hunk 4 above)
Comment (AlbertoCastellon) [reply src/app/cars/cars.component.html]: Some "leftover code" that doesn't belong here. Deleted.

**Review by javieraviles — APPROVED** [review]

**Pull request merged**